akiran/react-slick
# Infinite scroll bug

## Overview
When viewing the last slide of a gallery and clicking next the slide transitions left instead of transitioning right.

The same issue is documented here.
https://github.com/akiran/react-slick/issues/698

The issue is reproducible by setting slick settings to `lazyLoad` and `infinite` to true.

## Test

1. Run `npm start` to bring up demo
2. View Lazy Load example.
3. Verify you are able to scroll pass the last slide.
